pages/user/[username]/notifications.tsx

```diff
@@ -0,0 +1,230 @@
+    }
+  }, [sessionToken, username]);
+
+  const handlePageChange = (page: number) => {
+    setPage(page);
+    fetchNotifications(page);
+    if (topRef.current) {
+      topRef.current.scrollIntoView({
+        behavior: "smooth",
+      });
+    }
+  };
+
+  return (
+    <div className="w-full ">
+      <TopNav />
+      <div
+        className="container flex flex-col justify-between w-full px-2 pt-16 mx-auto overflow-hidden md:px-16 md:flex-row md:gap-20 lg:gap-40"
+        ref={topRef}
+      >
+        <div className="flex flex-col gap-4 w-80 ">
+          <Title className="!text-2xl !text-light-slate-12" level={3}>
+            Notifications
+          </Title>
+          <aside className="flex flex-col gap-2">
+            <Button
+              variant="link"
+              className={clsx("hover:text-orange-600", filter === "all" ? "text-orange-600" : "text-light-slate-11")}
+              onClick={() => {
+                setFilter("all");
+                router.push(`/user/${username}/notifications?filter=all`);
+              }}
+            >
+              All
+            </Button>
+            <Button
+              variant="link"
+              className={clsx("hover:text-orange-600", filter === "follow" ? "text-orange-600" : "text-light-slate-11")}
+              onClick={() => {
+                setFilter("follow");
+                router.push(`/user/${username}/notifications?filter=follow`);
+              }}
+            >
+              Follow
+            </Button>
+            <Button
+              variant="link"
+              className={clsx(
+                "hover:text-orange-600",
+                filter === "highlight_reaction" ? "text-orange-600" : "text-light-slate-11"
+              )}
+              onClick={() => {
+                setFilter("highlight_reaction");
+                router.push(`/user/${username}/notifications?filter=highlight_reaction`);
+              }}
+            >
+              Highlight Reaction
+            </Button>
+          </aside>
+        </div>
+        <div className="flex-1 mt-10 md:mt-0">
+          {loading ? (
+            <div className="flex flex-col gap-4">
+              {Array.from({ length: 8 }).map((_, index) => (
+                <div className="flex gap-2" key={index}>
+                  <SkeletonWrapper width={50} height={50} />
+                  <div className="">
+                    <SkeletonWrapper height={20} width={500} classNames="mb-2" />
+                    <SkeletonWrapper height={10} width={100} />
+                  </div>
+                </div>
+              ))}
+            </div>
+          ) : notifications?.length <= 0 ? (
+            <DashContainer>
+              <div className="text-center">
+                <p>
+                  You don&apos;t have any{" "}
+                  {changeCapitalization(filter === "highlight_reaction" ? "Highlight Reactions" : filter, true)}{" "}
```

Comment:
```suggestion
                  {changeCapitalization(filter === "highlight_reaction" ? "reaction" : filter, true)}{" "}
```
grammar is better this way.